WordPress/Tests/PHP/EnqueuedCheckUnitTest.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
```

Comment:
What about cases for a couple of the functions that have a version but no in_footer param?